feat(docs): add SSR and SSG docs (#4287)
twilio-labs/paste: packages/paste-website/src/pages/theme/changing-theme.mdx

**For other themes, see our [manual installation page](/introduction/for-engineers/manual-installation/#how-to-load-the-right-font) for more information.**

## SSR and SSG

> If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching you will need to use CSS variables and setting a body data-theme attribute instead to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, dicussing the problem and demonstrating how we solved it in the Paste website. 

Reviewer: ```suggestion
If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching you will need to use CSS variables and setting a body data-theme attribute instead to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, discussing the problem and demonstrating how we solved it in the Paste website. 
```

> If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching, you will need to use CSS variables and setting a body data-theme attribute instead to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, discussing the problem and demonstrating how we solved it in the Paste website. 

Reviewer 2: ```suggestion
If you are using server-side rendering (SSR) or static site generation (SSG), <strong>and</strong> you support theme switching, you will need to use CSS variables and set a `body` `data-theme` attribute to avoid theme flicker on initial renders. You can view our <Anchor href="/blog/2025-03-20-css-variables-ssr-ssg/">blog post</Anchor> on this topic, discussing the problem and demonstrating how we solved it for the Paste website. 
```
